package/MDAnalysis/core/groups.py

```diff
@@ -975,6 +980,15 @@ class AtomGroup(GroupBase):
     .. SeeAlso:: :class:`MDAnalysis.core.universe.Universe`
 
     """
+    def __getitem__(self, item):
+        # u.atoms['HT1'] access, otherwise default
+        if isinstance(item, string_types):
```

Comment:
I do not see this behavior in the docstring above. Is this documented somewhere?